### ISSUE TITLE
(D3 - Histogram): A visual representation of emission attribute(s)

### ISSUE THREAD

**Issue by h181180** [kyma-as/api-dashboard#20]
**Problem description**
A graph, chart or some other **visualization of the emission attribute(s)** which will be a component that can be called later (button, auto-display, etc.). This component needs to generic such that it can be called on different attributes.

**Possible solution**
We need a graph or another visual representation of the emission attribute(s) for a specific vessel. Experiment with different charts and colors which can showcase this data in a nice and clear way. *This is supposed to be in a separate 'vessel view', not on the home screen/dashboard, but this is not important for the completion of this issue.*

**Additional info**
The 'vessel view' is another issue currently being worked on. The home screen will provide a set of *vessels* to click on which will redirect into this vessel view. Once a solution for this issue is finalized it will be ported/pasted into this 'vessel view' as part of its 'dashboard'. This vessel view will contain different visualizations of different attributes specific to that vessel.
In other words, from the home screen you click on one of the vessels and you're redirected to that vessel's dashboard/view. This dashboard contains a set of visualization components (like this issue) and ways to call/hide them, etc.

***Claim this issue before working on it, and then make your branch.***
